mlir/utils/jenkins/Jenkinsfile

```diff
@@ -92,9 +92,9 @@ pipeline {
     agent none
     parameters {
         // Below should be set statically by Jenkins job
-        booleanParam(name: 'nightly', defaultValue: params.nightly ?: false,
+        booleanParam(name: 'nightly', defaultValue: params.nightly ? true : false,
                      description: 'Run extra nightly-only tests')
-        booleanParam(name: 'canXdlops', defaultValue: params.canXdlops ?: true,
+        booleanParam(name: 'canXdlops', defaultValue: params.canXdlops == false ? false : true,
```

Comment:
Why does canXdlops' default value have to be false? I think the PR tests on private CI requires canXdlops to be true?

Reply:
This makes the default value true (I'm pretty sure). That is, if params.canXdlops is false, the expression is false, otherwise it is true.

Reply:
We can double-check what the PR tests do with this one though to be safe.

Reply:
Sounds look.

Reply:
The PR tests look correct, but the pre-merge checks caught errors. Have to fix them first.

Reply:
Seems to have been a transient issue. @yiqian1